packages/inference/test/HfInference.spec.ts

```diff
@@ -649,7 +645,89 @@ describe.concurrent(
 			const { generated_text } = await ep.textGeneration({
 				inputs: "one plus two equals",
 			});
-			expect(generated_text).toEqual("three");
+			assert(generated_text === "three" || generated_text === "3");
+		});
+
+		it("textGeneration - OpenAI Specs", async () => {
+			const ep = hf.endpoint(
+				"https://api-inference.huggingface.co/models/mistralai/Mistral-7B-Instruct-v0.2/v1/chat/completions"
+			);
+			const res = await ep.textGeneration({
+				model: "tgi",
+				messages: [{ role: "user", content: "Complete the this sentence with words one plus one is equal " }],
+				parameters: {
+					max_tokens: 500,
+					return_full_text: false,
+					temperature: 0.0,
+					seed: 0,
+				},
```

Comment:
for chat completion, parameters shoud be sent at the root level

```js
				model: "tgi",
				messages: [{ role: "user", content: "Complete the this sentence with words one plus one is equal " }],
				max_tokens: 500,
				return_full_text: false,
				temperature: 0.0,
				seed: 0,
```

Reply:
good catch , thanks